journal/modernized/01-journal.adoc

```diff
@@ -3496,7 +3496,7 @@ I was willing to be discharged, having been there part of three days.
 I told the bishop of good old David, who said, Ps. 26:6,
 "`I will wash my hands in innocency, so will I compass your altar, O God;
 that I may publish with the voice of thanksgiving and tell of all your wondrous works.`"
-Then I said to the bishop.
+Then I said to the bishop,
 You canst not say you will wash your hands in innocency, nor compass the altar of God,
```

Comment:
Just noticed this, I wonder if you could sneak it into your corrections, this line should read "You can", not "You canst" -- this would be an error in the automated modernizing that Jason did.